test/sql/test_window_function/R/test_cume_window_function

```diff
@@ -0,0 +1,705 @@
+-- name: test_next_day
+DROP TABLE IF EXISTS `cume_test`;
+-- result:
+-- !result
+CREATE TABLE IF NOT EXISTS `cume_test` (
+  k1 INT,
+  k2 INT,
+  v1 INT SUM,
+  v2 DOUBLE MAX,
+  v3 DATE MIN,
+  v4 VARCHAR(20) REPLACE
+)
+ENGINE = olap
+AGGREGATE KEY(k1, k2)
+DISTRIBUTED BY HASH(`k1`)
+PROPERTIES(
+  "replication_num" = "1"
+);
+-- result:
+-- !result
+INSERT INTO `cume_test` (k1, k2, v1, v2, v3, v4)
+VALUES (1, 10, 15, 2.5, '2023-01-01', 'apple'),
+       (1, 2, 15, 2.5, '2023-01-02', 'banana'),
+       (1, 8, 15, 3.5, '2023-01-03', 'cherry'),
+       (1, 7, 20, 3.5, '2023-01-04', 'date'),
+       (2, 6, 20, 3.5, '2023-01-05', 'fig'),
+       (2, 5, 20, 4.5, '2023-01-06', 'grape'),
+       (2, 4, 20, 5.5, '2023-01-07', 'honeydew'),
+       (3, 3, 20, 5.5, '2023-01-08', 'kiwi'),
+       (3, 2, 20, 5.5, '2023-01-09', 'lemon'),
+       (4, 2, 25, 5.5, '2023-01-10', 'mango'),
+       (4, 10, 30, 5.5, '2023-01-11', 'nectarine'),
+       (4, 9, 30, 5.5, '2023-01-12', 'orange'),
+       (4, 8, 35, 6.5, '2023-01-13', 'papaya'),
+       (4, 7, 35, 6.5, '2023-01-14', 'pineapple'),
+       (4, 6, 35, 7.5, '2023-01-15', 'quince'),
+       (4, 5, 35, 7.5, '2023-01-16', 'raspberry'),
+       (5, 4, 35, 8.5, '2023-01-17', 'strawberry'),
+       (5, 3, 40, 8.5, '2023-01-18', 'tangerine'),
+       (5, 2, 40, 8.5, '2023-01-19', 'ugli fruit'),
+       (5, 1, 40, 8.5, '2023-01-20', 'vanilla'),
+       (5, 10, 45, 9.5, '2023-01-21', 'watermelon'),
+       (6, 9, 45, 10.5, '2023-01-22', 'xigua'),
+       (6, 8, 50, 11.5, '2023-01-23', 'yumberry'),
+       (6, 7, 50, 12.5, '2023-01-24', 'zucchini'),
+       (6, 6, 55, 13.5, '2023-01-25', 'acerola'),
+       (6, 5, 60, 13.5, '2023-01-26', 'bilberry'),
+       (7, 4, 60, 13.5, '2023-01-27', 'cloudberry'),
+       (7, 3, 60, 14.5, '2023-01-28', 'dewberry');
+-- result:
+-- !result
+SELECT *, CUME_DIST() OVER (ORDER BY v1) AS cume_dist_v1 FROM cume_test;
```

Comment:
please add some cases without order-by clause, and the data with null values.

Reply:
Done.